config.yaml

```diff
@@ -768,6 +768,12 @@ repositories:
     name: tag-env-sustainability
     settings:
       has_wiki: true
+  - name: tag-env-sustainability-website
+    teams:
+      tag-env-sustainability: write
+      cncf-tech-docs: admin
+    settings:
+      has_wiki: true
```

Comment:
Does it need a wiki? (I tend to turn all the extras off till they're needed, wikis, discussions, sponsorship, etc.)

I'll defer to @leonardpahlke tho.

Reply:
We don't need a wiki.